tedana/resources/config/outputs.json

```diff
@@ -182,5 +182,9 @@
     "ICA orthogonalized mixing tsv": {
         "orig": "ica_orth_mixing",
         "bidsv1.5.0": "desc-ICAOrth_mixing"
+    },
+    "System info json": {
+        "orig": "system_info",
+        "bidsv1.5.0": "system_info"
```

Comment:
I think we can put all of this info into the `dataset_description.json` instead. See [here](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#derived-dataset-and-pipeline-description). We could also probably steal some field names from [task events metadata](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/05-task-events.html#stimulus-presentation-details). They wouldn't be completely BIDS-compliant, but they're going to be fairly close.

Reply:
Done in aded008